preservation-workflow.html

```diff
@@ -397,7 +438,8 @@ <h3>System to System Flow</h3>
         The Repository calls the Gateway <a href="otm-gateway.html#initiate-restore">POST Object Restore</a> endpoint for
         the object to be restored
       </li>
-      <li>The Gateway resolves the object into a set of files to be restored</li>
+      <li>The Gateway calls the Bridge <a href="otm-bridge.html#get-deposited-details">Get Deposited Content</a> in order to
```

Comment:
@no-reply I updated the Gateway/Bridge interaction here for the restore workflow to use https://ucsdlib.github.io/otm-specs/otm-bridge.html#get-deposited-details in order to get the files for a deposit. It seems like this should match what the latest iteration of our workflow has been.